lms/envs/common.py

```diff
@@ -413,7 +413,7 @@
     'ENABLE_OPENBADGES': False,
 
     # Credit course API
-    'ENABLE_CREDIT_API': False,
+    'ENABLE_CREDIT_API': True,
```

Comment:
We enable this setting using configuration overrides when deploying to stage and production. Do we want to enable it by default for our open source users?